src/kvstore-client/index.ts

```diff
@@ -12,6 +12,9 @@ export const ErrorIsWrongRevision = (error: ErrorWithCode): boolean => error.cod
 export const ErrorIsBadGeneration = (error: ErrorWithCode): boolean => error.code === KVStoreErrorType.BadGeneration
 export const ErrorIsNotFound = (error: ErrorWithCode): boolean => error.code === KVStoreErrorType.NotFound
 
+const entryValueIsSet = (entryValue: string | null | undefined): boolean =>
+  entryValue !== '' && entryValue !== null && entryValue !== undefined
```

Comment:
should we keep `entryValue !== ''` or not?

Reply:
i added this to the python library. yes, i think we should leave it to make the bot libraries a little more robust to client versioning.